src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -779,7 +779,45 @@ public IEnumerable<Element> GetJoinedElements()
                 .ToList();
         }
 
+        /// <summary>
+        /// Unjoins elements if they are joined 
+        /// </summary>
+        /// <param name="elements">List of elements to unjoin</param>
+        /// <returns>Elements that have been unjoined</returns>
+        public static List<Element> UnjoinGeometry(List<Element> elements)
```

Comment:
we should return either 
- a list of lists : for every element, we return a list of elements that were unjoined. this will be more useful than a single big flat list of stuff
- a dictionary : keys are each of the elements and values are the list of elements that were unjoined for each of them

Reply:
yea thought about that. The only concern is that we will probably return a bunch of duplicated elements in that way, as the same element could be joined to multiple other elements. If we, for example, input 10 elements we could potentially return each object x10 times. Maybe that isn't any concern?

You could pretty easily find the which elements have been unjoined from which element by using the `AreJoined` node.
![image](https://user-images.githubusercontent.com/13732445/69672802-c4202f00-1090-11ea-989f-3b1eaf71e69f.png)

Reply:
hm, good points - though still think the nested lists will be more useful and Dynamo-like than an already flattened one, reducing the need for `FilterByBoolMask` and other things that we've seen are hard for people to get right. Let's discuss in the morning, think i might have the inputs/use-case a bit wrong in my head.

Also, do we need to rename this node to `UnjoinAllGeometry` as agreed with Sol and add another one that unjoins specific elements ?

Reply:
@radumg made this `UnjoinAllGeometry` and made it return sublists. Do you want me to add a node that unjoins specific elements to the list?

Reply:
yes please ! let's add one that `UnjoinGeometry` with `element` and `otherElement` as inputs, like one of the other nodes